mteb/models/e5_instruct.py

```diff
@@ -46,7 +46,11 @@ def encode(
                 )
             if instruction:
                 kwargs["instruction"] = e5_instruction(instruction)
-            return super().encode(sentences, *args, **kwargs)
+            embeddings = super().encode(sentences, *args, **kwargs)
+            if isinstance(embeddings, torch.Tensor):
+                # sometimes in kwargs can be return_tensors=True
+                embeddings = embeddings.cpu().detach().float().numpy()
```

Comment:
Should we move this normalization somewhere else? (I believe it used to be in the model_encode function, but it might be nice to have a normalization function.

Reply:
I don't know where to put it. Most of the tasks return numpy embeddings and this can be removed, but in STS somehow passed `return_tensors=True`. I tried to debug it, but couldn't find from where it passes

Reply:
hmm will try do a debug on this tomorrow or monday

Reply:
I found it #1325!